### PR TITLE
command/app: add --no-sign-request global flag to access public buckets

### LINKED ISSUE #285: Access to public buckets
Hi currently I access a public bucket through asw cli with  --no-sign-request parameter.
I can not get s5cmd to replicate this behaviour it always fails with 403 permission denied

the bucket: 
s3://noaa-gfs-bdp-pds/

https://registry.opendata.aws/noaa-gfs-bdp-pds/

### PR DESCRIPTION
**Pull request by sonmezonur**
Resolves #285 

Tested it by using public bucket mentioned in issue:

Screen Shot 2021-06-25 at 18 27 15

### CODE REVIEW

File: command/app.go [hunk 1]
```diff
@@ -64,6 +64,10 @@ var app = &cli.App{
 			Name:  "stat",
 			Usage: "collect statistics of program execution and display it at the end",
 		},
+		&cli.BoolFlag{
+			Name:  "no-sign-request",
+			Usage: "do not sign requests",
```
Comment (seruman) [inline command/app.go:69]: should we indicate that the credentials won't be loaded?

**Review by seruman — COMMENTED** [review]
could you update changelog?

File: command/app.go (hunk 1 above)
Comment (sonmezonur) [reply command/app.go]: updated both help text and changelog

**Review by seruman — DISMISSED** [review]

**Comment by igungor** [comment]
Could you add an item to the `ls` examples list demonstrating access to a public bucket?

**Comment by sonmezonur** [comment]
> Could you add an item to the `ls` examples list demonstrating access to a public bucket?

Added examples for `ls` and `cp`

**Review by aykutfarsak — APPROVED** [review]

**Review by seruman — APPROVED** [review]

**Review by igungor — APPROVED** [review]

**Pull request merged**